### PR TITLE
Italian: Add translations for '# Requires translation' lines

### PR DESCRIPTION
**Pull request by TommasoPetrolito**
### List of edits on file `android/assets/jsons/translations/Italian.properties`

- **Line 10**: `StartWithCapitalLetter = true` _## in Italian, just like English, Spanish, French, sentences start with a capital letter_
- **Line 19 --> 18**: `Fastlane_short_description = Gioco 4X - Costruire una civiltà` _## this translation takes inspiration from French one, that approached the issue behind the super compact wording in an elegant way (in Italian, just like in French, using the noun for "building"/"costruzione" would not have worked as good as this solution)_
- **Line 24 --> 22**: `Fastlane_full_description = Una reinterpretazione open source del gioco di gestione delle civiltà più famoso di sempre; per di più veloce, leggero, senza pubblicità e gratuito per sempre!\n\nCostruisci il tuo impero, ricerca tecnologie, espandi le tue città ed elimina i tuoi avversari!\n\nSuggerimenti? Bug? L'elenco dei problemi aperti è disponibile nella <a href="https://github.com/yairm210/Unciv/issues">pagina Github - Issues</a>. Qualsiasi aiuto è benvenuto!\n\nQualche domanda? Commenti? O anche solo molto tempo libero da investire? Vieni a chattare con noi sul <a href="https://discord.gg/bjrB4Xw">server Discord Unciv</a>.\n\nVuoi dare una mano a tradurre il gioco nella tua lingua? Segui le istruzioni nell'<a href="https://yairm210.github.io/Unciv/Other/Translating">articolo wiki</a> per i traduttori.\n\nKotlin e/o Java non hanno segreti per te? Sentiti libero di partecipare allo sviluppo sulla <a href="https://github.com/yairm210/Unciv">pagina Github</a>.\n\nIl mondo ti sta aspettando! Costruirai una civiltà in grado di superare la prova del tempo?\n\nL'autorizzazione "abilita l'accesso completo alla rete" è richiesta per le seguenti funzionalità:\n- Elencare e scaricare le mod disponibili\n- Scaricare la musica e i suoni\n- Caricare e scaricare online lo stato delle partite multiplayer\nTutte le altre autorizzazioni elencate sono gestite automaticamente dall'API per la funzione di notifica del turno multiplayer.` _## this translation is quite direct (not too much creative), and follows more or less the same approach used by French translation_
- **Line 509 --> 506**: `Floodfill_Abbreviation = R` ## _this is not a translation: I put R due to the Floodfill value at Line 551 --> 548 that is: "Riempi". Comparing the approach used in Spanish and French solutions (respectively C for "Cubeta" in Spanish and R for "Remplir" in French) it seams like the starting capital letter is used as Abbreviation, I might be wrong so if this is not the expected behavior/approach please just fill free to add more information about this point_
- **Line 815--> 811**: `Font family = Tipo font` _## this translation is similar to the Spanish one and means "Font Type". In Italian we don't use the "family" equivalent "famiglia" in this context so "tipo" (meaning "type") is more generic but works well. In Italian, compared to Spanish and French even just leaving "Font Family" or just "Font" in English might work well because Informatics/digital editing concepts are often expressed directly in English._
- **Line 1016 --> 1011**: `EffectBeforeCause = true` _## after reading the detailed explanation in the code-comments above this line, I can just simply say that in Italian, with the current version of the translations, this EffectBeforeCause = true makes sense as well as in English and other languages_
- **Line 1796 --> 1791**:  
OLD LINES
    `- Must be next to [terrain] = La città deve rasentare [terrain] `
    `- Must be on [terrain] = Deve ritrovarsi su [terrain] `
NEW LINES
    `+ Must be next to [terrain] = Deve confinare con [terrain] `
    `+ Must be on [terrain] = Deve stare su [terrain] `
   _## just minor changes following the Spanish approach for global consistency and avoiding references that were not present in the original English text. E.g. "città" ("city") is not present in the original text and, depending on the final context, using it inside this sentence might result in a redundant unwanted repetition of this word_
- **Line 1834 --> 1828**: `ConditionalsPlacement = after` _## this makes sense with Italian as well, with the current version of translations_
- **Lines 2379 and following ones**: I have found inconsistency and confusion about this part, also attempting to compare with Spanish or French approach (I had to check because I'm not really sure where these texts will be shown in the final context) I have not found any clear unambiguous way to approach these. These portions of text look like variable names and even follow a strict **CamelCase** formatting. It is not clear if also their translations should follow this CamelCase format or not. In Spanish and French there is no consistency, some are CamelCase and some are not. In this code-change proposal for Italian I always put separated words using white spaces instead of using CamelCase formatting, but if I got it wrong, please let me know and add any useful information about this point @yairm210 (not sure if tagging you makes sense, sorry if this is not the expected process, this is my first PR in this project and yet I am not confident about which Discord server sub channels are the correct places to discuss or point out this kind of topics).

Other changes, including new files and folders, are just the result of generating the language files (now there is an `it/` folder inside `fastlane/metadata/android/`) `full_description.txt` and `short_description.txt` have been created and the translation percentage for Italian rose up from 99% to 100% inside `android/assets/jsons/translations/completionPercentages.properties`, while I'm not sure why generating the language files moved Simplified Chinese in a different line number (from 23 to 24 switching with the line of Bulgarian) but the information kept identical as before.

For some reasons also `fastlane/metadata/android/pt/short_description.txt` got updated with the Portuguese Short Description that anyway makes sense to me.

I'm not sure if these generated language metadata files should be tracked under GIT or not (if we don't want them to be tracked maybe we can just add some exceptions inside the `.gitignore` file).

Thank you very much in advance for any feedback you can provide!

### CODE REVIEW

**Comment by alexban011** [comment]
Welcome to the community. 

You really don't need to spend time on writing what you changed. Git already shows this to the reviewers as seen here https://github.com/yairm210/Unciv/pull/9915/commits/2ae3078bbb4ae133417111dfb5b0fd0fa8889706

**Comment by TommasoPetrolito** [comment]
> Welcome to the community.
> 
> You really don't need to spend time on writing what you changed. Git already shows this to the reviewers as seen here [2ae3078](https://github.com/yairm210/Unciv/commit/2ae3078bbb4ae133417111dfb5b0fd0fa8889706)

Thank you, and sorry for the verbosity, being this my first change and not knowing the final context I wanted to add a few details about the reasoning/approach that I applied.
Do you have hints about how to proceed?

- The CamelCase translations if I got it right (from discord #translation channel) are not actually shown anywhere, so maybe can can be kept like I drafted I assume, stated that they are not actually useful, but if we want to choose a conventional way to translate these anyway I can still change them. 

- I still have some doubts about the Floodfill_Abbreviation for which I simply put the starting capital letter of the already present Floodfill variable (R for Riempi), does this make sense?

If everything is already OK, I think I can promote the draft PR to "ready for review"? Any drawbacks?

**Comment by yairm210** [comment]
All the CamelCase 'translations' are not really translations, but sort of configurations for translations, and each of them should have a comment explaining what they do
If you find one that doesn't have a comment, please tell me :)
Only exceptions I can think of are the unit types (MeleeWater etc)

**Comment by TommasoPetrolito** [comment]
> All the CamelCase 'translations' are not really translations, but sort of configurations for translations, and each of them should have a comment explaining what they do If you find one that doesn't have a comment, please tell me :) Only exceptions I can think of are the unit types (MeleeWater etc)

Thank you very much! Happy to have contributed.
I put a specific review-comment in the delta of changes so that you can have a direct look to that section of CamelCase configurations/variables (UnitTriggerable, UnitTriggerCondition and so on). Maybe adding a code-comment just above that part can help translators in the future.

File: android/assets/jsons/translations/Italian.properties [hunk 1]
```diff
@@ -2383,8 +2376,7 @@ all healing effects doubled = effetti curativi raddoppiati
 The Spaceship = L'Astronave
 Maya Long Count calendar cycle = Ciclo del Conto Lungo del calendario maya
 Triggerable = Attivabile
```
[the remark this answers is not in the document]
Comment (yairm210) [reply android/assets/jsons/translations/Italian.properties:2378]: No, these all look good :)

**Comment by TommasoPetrolito** [comment]
>You will be able to see your result after several delays on https://f-droid.org/it/packages/com.unciv.app/ - that's where they are meant to end up. Yes, the files under `fastlane/metadata` are interim stages - we didn't want a gitignore, having them tracked is fine, but _you_ don't need to, and whether you push them or not is unimportant, the conversion is run during the release workflow anyway. You can use your private `.git/info/exclude` instead of the project-shared `.gitignore` if you want... 

Thank you for the suggestion! It makes perfectly sense

> 
> > Floodfill_Abbreviation
> 
> ... ends up here
> And it's inspired by image
> 
> * so if you got Gimp installed in italiano you could look there. I might have expected 'inondare' 🤪

Thank you! That tool in GIMP is named "Riempimento Colore" so I assume the "Riempi"/"R" makes sense. I have found the Floodfill variable already valued to "Riempi" so my doubt was mainly about what rule was behind selecting a specific character from the word "Riempi" for the Floodfill_Abbreviation and I assumed the first capital letter was the best option.

Thank you again.

**Pull request merged**